source/_components/switch.rainmachine.markdown

```diff
@@ -2,15 +2,15 @@
 layout: page
 title: "RainMachine Switch"
 description: "Instructions on how to use RainMachine units with Home Assistant."
-date: 2017-08-04 12:00
+date: 2017-08-14 13:30
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: rainmachine.png
 ha_category: Switch
 ha_iot_class: "Cloud Polling"
-ha_release: 0.51
+ha_release: 0.52
```

Comment:
Sorry, I'm not sure what I was thinking, but since this is a platform already released in 0.51, this value shouldn't be changed at all.